### PR TITLE
Add a standard gitignore across all projects

### PR DESCRIPTION
**Pull request by samuelallan72**
Fixes: #80

### CODE REVIEW

**Comment by samuelallan72** [comment]
I'm interested to hear your thoughts on this - a common gitignore file for all our projects.  I've combined what I found across several projects, and removed some cruft.  Please check for things that might be missing. :)

File: terraform-plans/templates/github/gitignore.tftpl [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+deb_dist/
+dist/
+*.egg-info/
+
+# Log files
+*.log
+
+# IDEs
+.idea/
+.vscode/
```
Comment (gabrielcocenza) [inline terraform-plans/templates/github/gitignore.tftpl:42]: nit:
I think this kind of gitignore it's responsibility of the developer to set a global ignore locally. There are too many possible IDEs to have an exhaustive list

File: terraform-plans/templates/github/gitignore.tftpl [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+# vi/vim swap files
+*.swp
```
Comment (gabrielcocenza) [inline terraform-plans/templates/github/gitignore.tftpl:45]: This too...

File: terraform-plans/templates/github/gitignore.tftpl (hunk 1 above)
Comment (samuelallan72) [reply terraform-plans/templates/github/gitignore.tftpl:42]: Yep I 100% agree :+1:  If there's no concern from others, I'm very happy to remove them.

**Review by chanchiwai-ray — DISMISSED** [review]

File: terraform-plans/templates/github/gitignore.tftpl (hunk 1 above)
Comment (samuelallan72) [reply terraform-plans/templates/github/gitignore.tftpl:42]: Updated :)

File: terraform-plans/templates/github/gitignore.tftpl (hunk 2 above)
Comment (samuelallan72) [reply terraform-plans/templates/github/gitignore.tftpl:45]: Updated :)

**Review by chanchiwai-ray — APPROVED** [review]

**Review by gabrielcocenza — APPROVED** [review]

**Pull request merged**